project/plugins.sbt

```diff
@@ -17,14 +26,13 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
 addSbtPlugin("com.typesafe.sbt" % "sbt-gzip" % "1.0.2")
 addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.12")
 
-// S3 based SBT resolver
-
-resolvers += "HAT Library Artifacts Snapshots" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com"
 
 addSbtPlugin("org.hatdex" % "sbt-slick-postgres-generator" % "0.0.11")
 
 // run "sbt dependencyUpdates" to check maven for updates or "sbt ";dependencyUpdates; reload plugins; dependencyUpdates" for sbt plugins
 addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.4")
 
 // ScalaFMT
-addSbtPlugin("org.scalameta" %% "sbt-scalafmt" % "2.4.0")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.3.4")
+
+addSbtPlugin("io.dataswift" % "sbt-scalafmt-common" % "0.1.1-SNAPSHOT")
```

Comment:
Does scala fmt has a rule for auto-inserting new line at EOF?

Reply:
It's on by default, and in fact, you can't turn it off.

Go UNIX!